src/Tester/packages.config

```diff
@@ -1,10 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Bond.Runtime.CSharp" version="3.0.7" targetFramework="net45" />
   <package id="Microsoft.Data.Edm" version="5.6.0" targetFramework="net45" />
   <package id="Microsoft.Data.OData" version="5.6.0" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.6.0" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="2.0.0" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="5.0.8" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="6.0.5" targetFramework="net45" />
```

Comment:
Newtonsoft.Json is also used by other projects - OrleansAzureUtils and OrleansZooKeperUtils. If we need to upgrade, let's upgrade them all, so that we use a single version.